### PR TITLE
Add unit tests for PageListItemLabelsUseCase

### PR DESCRIPTION
**Pull request by malinajirka**
Adds unit tests for https://github.com/wordpress-mobile/WordPress-Android/pull/11278

Review instruction
1. Review the code
2. Wait until https://github.com/wordpress-mobile/WordPress-Android/pull/11280 gets merged
3. Change target branch to feature/master-pages-offline-support
4. Remove the "Not ready for merge" label
5. Merge the PR

PR submission checklist:

- [x] I have considered adding unit tests where possible.
- [x] I have considered adding accessibility improvements for my changes.
- [x] I have considered if this change warrants user-facing release notes and have added them to `RELEASE-NOTES.txt` if necessary.

### CODE REVIEW

File: WordPress/src/test/java/org/wordpress/android/viewmodel/pages/CreatePageListItemLabelsUseCaseTest.kt [hunk 1]
```diff
@@ -0,0 +1,365 @@
...
+    @Test
+    fun `pending review label shown for pages pending review`() {
+        val (labels, _) = useCase.createLabels(
+                PostModel().apply { setStatus(PENDING.toString()) },
+                mock()
+        )
+        assertThat(labels).contains(UiStringRes(R.string.page_status_pending_review))
+    }
+
+    //
```
Comment (jd-alexander) [inline WordPress/src/test/java/org/wordpress/android/viewmodel/pages/CreatePageListItemLabelsUseCaseTest.kt]: Were you planning to put a comment here? Let me know 😄

Comment (malinajirka) [reply WordPress/src/test/java/org/wordpress/android/viewmodel/pages/CreatePageListItemLabelsUseCaseTest.kt]: Ohh, I missed that :P - Fixed in e8ce3a0cdb03c96fb66f2af70156f70148fefe3b

**Comment by malinajirka** [comment]
>It works as intended! Based on the comments I made on the page item labels PR, this one might get updated with more tests depending on your thoughts. So once you respond about to this, we can move forward here.

Tests updated in e8ce3a0cdb03c96fb66f2af70156f70148fefe3b. I decided to replace the static access to UploadUtils with an injectable wrapper. These tests are supposed to be unit test testing only `CreatePageListItemLabelsUseCase` I decided to remove some of the tests.

I created a ticket https://github.com/wordpress-mobile/WordPress-Android/issues/11285 - for adding unit tests for the UploadUtils method.

It's ready for another round, thanks!

**Comment by jd-alexander** [comment]
Thanks for these changes @malinajirka! Looks good! 

For some reason when I am running the test locally I am getting this exception. 

```
org.mockito.exceptions.base.MockitoException: 
Mockito cannot mock this class: class org.wordpress.android.ui.uploads.UploadUtilsWrapper.
Can not mock final classes with the following settings :
 - explicit serialization (e.g. withSettings().serializable())
 - extra interfaces (e.g. withSettings().extraInterfaces(...))

You are seeing this disclaimer because Mockito is configured to create inlined mocks.
You can learn about inline mocks and their limitations under item #39 of the Mockito class javadoc.
```

I am going to look into my setup to see if it might be causing the issue.

**Comment by malinajirka** [comment]
>For some reason when I am running the test locally I am getting this exception.

That's quite weird. I have no idea what might be causing this issue. I've double checked it and I don't see such exception on my end.

**Pull request merged**